package.json

```diff
@@ -308,10 +308,15 @@
 		"typescript": "^3.9.7",
 		"vsce": "^1.61.0",
 		"webpack": "^5.5.0",
-		"webpack-cli": "^4.2.0"
+		"webpack-cli": "^4.2.0",
+		"@types/rimraf": "^2.0.2",
+		"rimraf": "^2.6.3",
+		"@types/tmp": "^0.1.0",
+		"tmp": "^0.1.0",
+		"temp-dir": "2.0.0"
```

Comment:
why? you are not really need they. just mock the io operations where it needed.
please remove - we do not want to maintenance additional packages